components/brave_shields/browser/ad_block_component_installer.cc

```diff
@@ -199,6 +199,11 @@ void RegisterAdBlockDefaultResourceComponent(
       cus, base::BindOnce(&OnRegistered, kAdBlockResourceComponentId));
 }
 
+void CheckAdBlockDefaultResourceComponentUpdate() {
+  BraveOnDemandUpdater::GetInstance()->OnDemandUpdate(
+      kAdBlockResourceComponentId);
```

Comment:
It would be good to add the default list components here as well (there are already constants defined for them, but for reference they are `iodkpdagapdfkphljnddpjlldadblomo` and `adcocjohghhfpidemphmcmlmhnfgikei`)

Reply:
You mean that we want to adjust update interval also for `kAdBlockExceptionComponentId` and `kAdBlockDefaultComponentId`?

Reply:
Updated https://github.com/brave/brave-core/pull/20557/commits/0979af19914c0fad23b3388a5bb6b0ff9feecc87